api-schema/src/main/java/org/stellar/anchor/api/sep/sep31/Sep31GetTransactionResponse.java

```diff
@@ -44,11 +45,8 @@ public static class TransactionResponse {
     @SerializedName("amount_out_asset")
     String amountOutAsset;
 
-    @SerializedName("amount_fee")
-    String amountFee;
-
-    @SerializedName("amount_fee_asset")
-    String amountFeeAsset;
+    @SerializedName("fee_details")
+    FeeDetails feeDetails;
```

Comment:
Was it missing in the response? Can you port addition of this field back to v2? And can you also please add a test to check that feeDetails is present in SEP6/24/31 responses? (I think I added them but I might've missed some)

Reply:
Yeah it was missing in SEP31 response. Will double check on the tests.